packages/component-library/src/components/plmg-radio-button/plmg-radio-button.scss

```diff
@@ -0,0 +1,127 @@
+@use '~@ducky/plumage-tokens/dist/scss/default' as tokens;
+
+// Component style
+// -------------------------------------------------
+.radio-button-wrapper {
+  display: flex;
+  cursor: pointer;
+  align-items: center;
+  width: fit-content;
+  border-radius: tokens.$plmg-border-radius-s;
```

Comment:
I believe border-radius should be 2px, which is xs

![image](https://user-images.githubusercontent.com/55230735/169550762-1dd7b212-767d-42a9-a065-9d397f5cfd80.png)